### PR TITLE
ticket 4, fetches all articles. some styling completed.

### PR DESCRIPTION
**Pull request by neisuns**

### CODE REVIEW

File: src/components/ArticleCard.jsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+const ArticleCard = ({ article }) => {
+    const {author, created_at, title, topic } = article;
+    const myTopic = topic.charAt(0).toUpperCase() + topic.slice(1);
+    const myDate = created_at.substring(0, 10);
+
+    return (<BrowserRouter>
+        <section className="ArticleCard">
+            <Link to={`/articles/${article.article_id}`}>
+                <h1>{article.title}</h1>
```
Comment (k-huggs) [inline src/components/ArticleCard.jsx]: You don't want multiple h1s on a single page

File: src/components/Articles.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+            setIsLoading(false);
+        })
+    }, [])
+
+    if (isLoading) {
+        return <p>Loading... Please wait</p>;
+    } 
+    return (
+        <div>
+            <h1>Articles</h1>
```
Comment (k-huggs) [inline src/components/Articles.jsx]: Careful of another h1!

**Review by k-huggs — COMMENTED** [review]
Have a look at making this more responsive as you build it out

File: src/components/Articles.jsx (hunk 2 above)
Comment (neisuns) [reply src/components/Articles.jsx]: thanks Kyle! I'll get these fixed :)

**Pull request merged**